src/lm.jl

```diff
@@ -156,7 +156,30 @@ function coeftable(mm::LinearModel)
               ["x$i" for i = 1:size(mm.pp.X, 2)], 4)
 end
 
-predict(mm::LinearModel, newx::Matrix) =  newx * coef(mm)
+# Different methods of prediction for the different factorization types
+
+predict(mm::LinearModel, newx::AbstractMatrix) = newx * coef(mm)
+
+function predict(mm::LinearModel, newx::AbstractMatrix, interval_type::Symbol, level = 0.95)
```

Comment:
Could you add a docstring to document what `interval_types` values are possible for `LinearModel` and what they mean?

It would also be great to add a generic docstring for `predict` [in StatsBase](https://github.com/JuliaStats/StatsBase.jl/blob/af2c1279b55c179bc3130203d1bf3614ca23ccf4/src/statmodels.jl#L153), to standardize on the order of arguments. The description of `newx`, which is common to all model types, should live there.